src/js/HomeLink.jsx

```diff
@@ -4,7 +4,10 @@ var Link = require('react-router').Link;
 var HomeLink = React.createClass({
 
   render: function(){
-    return <Link className="HomeLink" to='/'>Home</Link>
+    return <div class="AppContainer">
```

Comment:
I _think_ I originally did this and now it looks odd. No worries on this PR though, I'll fix when we move to USDS in #155